apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -18,94 +21,100 @@ function showHomepage() {
   const userId = homepageData.userid;
   const showInitialTips = !homepageData.initialtipsdismissed;
 
+  registerReducers({urlHelpers});
+  const store = getStore();
+  store.dispatch(setPegasusHost(homepageData.codeorgurlprefix));
+
   ReactDOM.render (
-    <div>
```

Comment:
All that happened here is that we wrapped the whole render in a `<Provider>`.  This file's diff in particular may be [easier to view with whitespace ignored](https://github.com/code-dot-org/code-dot-org/pull/16678/files?w=1)